frontend/public/components/build-config.tsx

```diff
@@ -89,13 +89,13 @@ const BuildConfigsHeader = props => <ListHeader>
 </ListHeader>;
 
 const BuildConfigsRow: React.SFC<BuildConfigsRowProps> = ({obj}) => <div className="row co-resource-list__item">
-  <div className="col-xs-3">
+  <OverflowYFade className="col-xs-3">
```

Comment:
I wonder if we shouldn't let names wrap. They can be important.

Reply:
Minimally we need to make sure there's at least a `title` attribute for `Overflow` / `OverflowYFade` so you can see the full name on hover.